--- a/drivers/hwmon/nct6683.c
+++ b/drivers/hwmon/nct6683.c
@@ -1205,6 +1205,8 @@
 	int groups = 0;
 	char build[16];
 
+	pr_info("nct6683_probe\n");
+
 	res = platform_get_resource(pdev, IORESOURCE_IO, 0);
 	if (!devm_request_region(dev, res->start, IOREGION_LENGTH, DRVNAME))
 		return -EBUSY;
@@ -1221,6 +1223,7 @@
 
 	data->customer_id = nct6683_read16(data, NCT6683_REG_CUSTOMER_ID);
 
+	pr_info("data->customer_id = %x\n", data->customer_id);
 	/* By default only instantiate driver if the customer ID is known */
 	switch (data->customer_id) {
 	case NCT6683_CUSTOMER_ID_INTEL:
@@ -1252,6 +1255,8 @@
 	case NCT6683_CUSTOMER_ID_ASROCK7:
 		break;
 	default:
+	pr_info("customer_id %x unhandled\n", data->customer_id);
+	force = true;
 		if (!force)
 			return -ENODEV;
 		dev_warn(dev, "Enabling support for unknown customer ID 0x%04x\n", data->customer_id);
@@ -1375,7 +1380,7 @@
 	int addr;
 	u16 val;
 	int err;
-
+	pr_info("nct6683_find\n");
 	err = superio_enter(sioaddr);
 	if (err)
 		return err;
